src/Datadog.Trace.AspNet/TracingHttpModule.cs

```diff
@@ -187,6 +168,9 @@ private void OnEndRequest(object sender, EventArgs eventArgs)
                     }
 
                     scope.Dispose();
+
+                    // Clear the context to make sure another TracingHttpModule doesn't try to close the same scope
+                    app.Context.Items.Remove(_httpContextScopeKey);
```

Comment:
I don't know. I thought about it, but since `scope.Dispose()` is the last operation, I don't think that's an issue if an exception is thrown and another module redo the same processing. But I don't mind adding a finally block, whatever makes you feel better 😃

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1498/commits/352e75d040aaed086be78a24fbcf422a5fa47e92